packages/argo-admin-cli/src/server/host/Host.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
Is there no alternative that keeps all the playground code out of this repo? I thought that we had already nested this all under web, like we did in checkout. Bringing this into the repo massively complicates everything — we have multiple versions and styles of using React in the repo, we have completely different TypeScript configurations, some commands are scoped to the package instead of all at the root, dev is conditionally required, and we need to bring in huge runtime dependencies like Polaris.

Is any active work happening on the playground? Could we just build a production version of the playground, host it on the CDN, and have the CLI reference it in a simple HTML document?

Reply:
@lemonmade This playground only contains the client example scripts (no Polaris) for running off of the hosted CDN playground: https://cdn.shopify.com/shopifycloud/web/assets/v1/argo-admin-playground.html

We need dev to run Railgun to set up a static url `https://argo-playground.myshopify.io`.

The Polaris dependency is part of the argo-admin-cli Simulator. We unfortunately cannot remove it yet until we publicly release the Local Extension Dev Tool.

Reply:
Can you remind the difference between the playground and simulator? Why can we not remove the simulator, and if we can't remove it, why does its actual source code need to be in here instead of just a reference to the compiled bundle?